### DISCUSSION TITLE
Time and Weather tiles for dashboard view?

### DISCUSSION THREAD

**Discussion by TwystNeko** [OctoFarm/OctoFarm#595] [Implemented]
So having set up Octofarm, I have the dashboard up on a monitor on the wall, and it occurred to me - it'd be real handy to have a simple clock & date tile for the dashboard view. A weather tile might be out of scope, but it'd sure be nice to have just from a "wall appliance" concept.

**Comment by NotExpectedYet** [comment]
Clock tile is certainly doable! 
The weather tile would have to be a public api I gather, I know a few so I will have a look into grabbing the data and adding it.

**Comment by NotExpectedYet** [comment]
> 
> 
> So having set up Octofarm, I have the dashboard up on a monitor on the wall, and it occurred to me - it'd be real handy to have a simple clock & date tile for the dashboard view. A weather tile might be out of scope, but it'd sure be nice to have just from a "wall appliance" concept.

![image](https://user-images.githubusercontent.com/61137988/142130602-5fe0a01d-8e86-4a1e-8aee-97ce4361814d.png)

One done, will be in V1.2
